### PR TITLE
Reorganise examples for list

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/1051
[Problem] Main examples mixed in main menu
[Solution] Simplify main menu. Categorize items.

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

File: examples/wearable/UIComponents/contents/presentation-views/list/main/sub-text/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		<header>
+			<h2 class="ui-title">
+				Sub Text
+			</h2>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview">
+				<li>
+					<a href="sub-text.html">
+						Basic
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/presentation-views/list/main/sub-text/index.html]: Sub text.

File: examples/wearable/UIComponents/contents/presentation-views/list/main/sub-text/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+					</a>
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/presentation-views/list/main/sub-text/index.html:29]: Please also restore comment that more examples are needed. Without such comment, this looks wired to have a sub menu for one item only.

**Review by grzegorz-czajkowski — CHANGES_REQUESTED** [review]
Found two issues probably from copy/paste. PR works ok.

File: examples/wearable/UIComponents/contents/presentation-views/list/main/sub-text/index.html (hunk 2 above)
Comment (lslachciak) [reply examples/wearable/UIComponents/contents/presentation-views/list/main/sub-text/index.html:29]: done

File: examples/wearable/UIComponents/contents/presentation-views/list/main/sub-text/index.html (hunk 1 above)
Comment (lslachciak) [reply examples/wearable/UIComponents/contents/presentation-views/list/main/sub-text/index.html]: done

**Review by grzegorz-czajkowski — APPROVED** [review]
LGTM

**Pull request merged**